system-int-test/test/cmr/system_int_test/search/collection_keyword_search_test.clj

```diff
@@ -44,14 +44,23 @@
         p2 (dc/platform
             {:short-name "platform_SnA spoonA"
              :long-name "platform_LnA"
-             :characteristics [(dc/characteristic {:name "char1" :description "char1desc"})
-                               (dc/characteristic {:name "char2" :description "char2desc"})]
+             :characteristics 
```

Comment:
I was thinking since I'm not adding additional dc/collection, therefore I didn't add additional work for the future convert. The main difficulty is I had tried to convert this test and failed. it's a tricky one and takes a lot of work.